engine.py

```diff
@@ -123,6 +123,7 @@ def __setup_ui(self, callback):
         t = tk_shotgun.Task(self, callback)
         
         # start up our QApp now
+        QtGui.QApplication.setLibraryPaths([])
```

Comment:
Have you talked to @mtiudaeu ? He did a fix in the Shotgun Desktop so that the QT_PLUGIN_PATHS was never set when Desktop is running. Is this an error that happens when you are launching from the Java applet or browser plugin or when running the Shotgun Desktop in the background?